windows container image version not compatible on Azure DevOps

While running release pipeline on Azure DevOps I am getting this error:
```
2020-04-28T15:54:06.9382528Z Step 1/16 : FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1903 AS base
2020-04-28T15:54:07.5667119Z 3.1-nanoserver-1903: Pulling from dotnet/core/aspnet
2020-04-28T15:54:07.8491344Z Service 'ecommerce.api.products' failed to build: a Windows version 10.0.18362-based image is incompatible with a 10.0.17763 host
2020-04-28T15:54:07.9635109Z ##[error]Building ecommerce.api.products
2020-04-28T15:54:07.9656087Z ##[error]Service 'ecommerce.api.products' failed to build: a Windows version 10.0.18362-based image is incompatible with a 10.0.17763 host
2020-04-28T15:54:07.9739950Z ##[error]The process 'C:\ProgramData\Chocolatey\bin\docker-compose.exe' failed with exit code 1
```

--- a/ECommerce.Api.Customers/Dockerfile
+++ b/ECommerce.Api.Customers/Dockerfile
@@ -3,11 +3,11 @@
 #Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
 #For more information, please see https://aka.ms/containercompat
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1903 AS base
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1809 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1903 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1809 AS build
 WORKDIR /src
 COPY ["ECommerce.Api.Customers/ECommerce.Api.Customers.csproj", "ECommerce.Api.Customers/"]
 RUN dotnet restore "ECommerce.Api.Customers/ECommerce.Api.Customers.csproj"

--- a/ECommerce.Api.Orders/Dockerfile
+++ b/ECommerce.Api.Orders/Dockerfile
@@ -3,11 +3,11 @@
 #Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
 #For more information, please see https://aka.ms/containercompat
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1903 AS base
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1809 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1903 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1809 AS build
 WORKDIR /src
 COPY ["ECommerce.Api.Orders/ECommerce.Api.Orders.csproj", "ECommerce.Api.Orders/"]
 RUN dotnet restore "ECommerce.Api.Orders/ECommerce.Api.Orders.csproj"

--- a/ECommerce.Api.Products/Dockerfile
+++ b/ECommerce.Api.Products/Dockerfile
@@ -3,11 +3,11 @@
 #Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
 #For more information, please see https://aka.ms/containercompat
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1903 AS base
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1809 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1903 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1809 AS build
 WORKDIR /src
 COPY ["ECommerce.Api.Products/ECommerce.Api.Products.csproj", "ECommerce.Api.Products/"]
 RUN dotnet restore "ECommerce.Api.Products/ECommerce.Api.Products.csproj"

--- a/ECommerce.Api.Search/Dockerfile
+++ b/ECommerce.Api.Search/Dockerfile
@@ -3,11 +3,11 @@
 #Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
 #For more information, please see https://aka.ms/containercompat
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1903 AS base
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-nanoserver-1809 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1903 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-nanoserver-1809 AS build
 WORKDIR /src
 COPY ["ECommerce.Api.Search/ECommerce.Api.Search.csproj", "ECommerce.Api.Search/"]
 RUN dotnet restore "ECommerce.Api.Search/ECommerce.Api.Search.csproj"

